cicd/pipelines/installing-pipelines.adoc

```diff
@@ -36,6 +37,10 @@ include::modules/op-installing-pipelines-operator-using-the-cli.adoc[leveloffset
 
 include::modules/op-pipelines-operator-in-restricted-environment.adoc[leveloffset=+1]
 
+// Disabling automatic creation of RBAC resources 
+
```

Comment:
Suggestion for L50:

- From "For more information on using pipelines in a restricted environment see:" to "For more information on using pipelines in a restricted environment, see:" (added a comma)